Kickstarter-iOS/Features/FindFriends/Controller/FindFriendsViewController.swift

```diff
@@ -33,8 +33,8 @@ internal final class FindFriendsViewController: UITableViewController {
     self.viewModel.inputs.viewDidLoad()
   }
 
-  override func viewWillAppear(_ animated: Bool) {
-    super.viewWillAppear(animated)
+  override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
```

Comment:
Is `animated` ever true here? By changing this from `viewWillAppear` to `viewDidAppear`, if animated is true, the nav bar will animate away after the view is on screen.

Reply:
yes, the `animated` is `true`, so the nav bar is appear after the view is on screen, I attached a video:

https://github.com/user-attachments/assets/c628b925-83c6-4c09-aea2-9e175b4a0361